docs/ui/prepare-migration/dataflow-tuning.md

```diff
@@ -0,0 +1,105 @@
+
+Present under the Networking section of the form. 
+
+### VPC Network
+Specify the name of the VPC network to use. For private connectivity, specify both the VPC network and subnetwork. If no network and subnetwork is provided, the [default](https://cloud.google.com/dataflow/docs/guides/specifying-networks#specifying_a_network_and_a_subnetwork) network is used.
+
+Present under the Networking section of the form. 
+
+### VPC Subnetwork
+Specify the name of the VPC subnetwork to use. For private connectivity, specify both the VPC network and subnetwork. If no network and subnetwork is provided, the [default](https://cloud.google.com/dataflow/docs/guides/specifying-networks#specifying_a_network_and_a_subnetwork) network is used.
+
+Present under the Networking section of the form. 
+
+{: .highlight }
+
+SMT sets the IP configuration based on VPC network and subnetwork. If either network or subnetwork is provided (running inside a VPC), the public IPs are disabled (IPConfiguration is private). If neither are provided, the IP configuration is set to PUBLIC.
+
+### Max Workers
+Specify the max workers for the dataflow job(s). By default, set to 50.
+
+Present under the Performance section of the form. 
+
+### Number of Workers
+Specify the initial number of workers for the dataflow job(s). By default, set to 1.
+
+Present under the Performance section of the form. 
+
+### Machine Type
+The machine type to use for the job, eg: n1-standard-2. Use default machine type if not specified.
+
+Present under the Performance section of the form. 
+
+### Service Account Email
+Specify a custom service account email to run the job as. Uses the default compute engine service account if not specified. For more details, click [here](https://cloud.google.com/dataflow/docs/reference/pipeline-options#security_and_networking).
+
+### Additional User Labels
+Additional user labels to be specified for the job via a JSON string. Example: { "name": "wrench", "mass": "1kg", "count": "3" }.
+
+### KMS Key Name
+Name for the Cloud KMS key for the job. Key format is: `projects/my-project/locations/us-central1/keyRings/keyring-name/cryptoKeys/key-name`. Omit this field to use Google Managed Encryption Keys.
+
+## Preset Flags
+
+These flags are set by SMT by default and <b>SHOULD NOT BE</b> modified unless running Dataflow in a non-standard configuration. To edit these parameters, click the edit button in the form next to the preset flags header.
+
+![](https://services.google.com/fh/gumdrop/preview/misc/preset-flags.png)
+
+### Dataflow Project
+Specify the project to run the dataflow job in.
+
+### Dataflow Location
+Specify the region to run the dataflow job in. It is recommended to keep the region same as Spanner region for performance. Example: us-central1
+
+### GCS Template Path
+Cloud Storage path to the template spec. Use this to run launch dataflow with custom templates. Example: `gs://my-bucket/path/to/template`
```

Comment:
It might makes sense to add a link here to the dataflow page here so that folks can read up more in detail there if required.